config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
-  use_doorkeeper
   namespace :api,
-  defaults: { format: :json }, constraints: { subdomain: "api" }, path: "/"  do
+  defaults: { format: :json }, constraints: {}, path: "/"  do
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.